onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -68,6 +68,8 @@
         "^test_maxpool_2d_ceil",
         "^test_maxpool_2d_dilations",
         "^test_maxpool_2d_uint8",
+        "^test_maxpool_with_argmax_2d_precomputed_pads_cpu", // maxpool test will fail if built with --traning-enabled
+        "^test_maxpool_with_argmax_2d_precomputed_strides_cpu", // maxpool test will fail if built with --traning-enabled
```

Comment:
Are these exclusions only needed for a DNNL build? If so, can we find some other way to disable them to avoid losing test coverage?

Reply:
These should only be excluded for the DNNL build these are listed under the `current_failing_tests_DNNL`. My understanding was that this was used if the `--use_dnnl` build flag was specified. I will double check.

Reply:
I just verified that when the code is built without the `--use-dnnl` flag tests are run.  So they are only disabled for dnnl.

Reply:
Apologies for the noise! I overlooked the context that these were added under the key `"current_failing_tests_DNNL"`.

Reply:
> Apologies for the noise! I overlooked the context that these were added under the key `"current_failing_tests_DNNL"`.

All good.  it is easy to miss since you only see the changed lines. I would rather have a more complete review with some noise.